rc/default_simulation.toml

```diff
@@ -3,21 +3,22 @@
 title = "Simulation configuration"
 
 [mode]
+training_type = 'clean'  # gauss or list or clean
 quiet = true
 
 [paths]
-outpath = "./build/example_data/"
```

Comment:
Default `outpath` should not be a path on big-tank

Reply:
Set back to default as it was before.